contrib/kube-prometheus/assets/prometheus/rules/prometheus.rules.yaml

```diff
@@ -42,3 +42,30 @@ groups:
     annotations:
       description: Prometheus {{ $labels.namespace }}/{{ $labels.pod}} is not connected
         to any Alertmanagers
+  - alert: PrometheusTSDBReloadsFailing
+    expr: increase(prometheus_tsdb_reloads_failures_total[2h]) > 0
+    for: 12h
+    labels:
+      severity: warning
+    annotations:
+      description: '{{$labels.job}} at {{$labels.instance}} had {{$value | humanize}}
+        reload failures over the last four hours.'
+      summary: Prometheus has issues reloading data blocks from disk
+  - alert: PrometheusTSDBCompactionsFailing
+    expr: increase(prometheus_tsdb_compactions_failed_total[2h]) > 0
+    for: 12h
+    labels:
+      severity: warning
+    annotations:
+      description: '{{$labels.job}} at {{$labels.instance}} had {{$value | humanize}}
+        compaction failures over the last four hours.'
```

Comment:
Where does the `four hours` come from?

Reply:
whoops, you're right, seems like that was mistaken with the alerting rule below